.eslintrc

```diff
@@ -1,27 +1,6 @@
 {
-  "extends": ["airbnb", "prettier"],
-  "plugins": ["prettier"],
+  "extends": ["./node_modules/eslint-tools-m6web/.eslintrc"],
```

Comment:
cela devrait suffire `"extends": "eslint-tools-m6web"`

Reply:
I tried but it doesn't !
```shell
Cannot find module 'eslint-config-eslint-tools-m6web'